rs/sns/swap/tests/swap.rs

```diff
@@ -1359,7 +1361,7 @@ async fn test_finalize_swap_ok_matched_funding() {
                 }
             }
         })
-        .flat_map(|b| &b.neuron_parameters)
+        .flat_map(|b| b.neuron_recipes.clone().unwrap().neuron_recipes)
         .map(|neuron_distribution| neuron_distribution.controller.as_ref().unwrap().to_string())
```

Comment:
NCR, just need to understand.

So we first map something *to* neuron recipes, and then map *from* neuron distribution... What's going on?

(Might be obvious if I've looked into the type inference in an IDE, but not obvious while reading raw code)